corehq/apps/domain/forms.py

```diff
@@ -1901,9 +1902,13 @@ def __init__(self, account, domain, creating_user, data=None, *args, **kwargs):
                         css_class='btn btn-default',
                         css_id='show_emails'
                     ),
-                    crispy.HTML('<p class="help-block">%s</p>' %
-                                _('Useful when you want to copy contact emails')),
-                    css_class='col-sm-9 col-md-8 col-lg-6'
+                    crispy.HTML(
+                        format_html(
+                            '<p class="help-block">{}</i> ',
```

Comment:
Shouldn't the `</i>` tag be `</p>` ?
Also, what's the reason for making this change? Both have same outputs
```
In [3]: format_html('<p class="help-block">{}</p>', 'Useful when you want to copy contact emails')
Out[3]: '<p class="help-block">Useful when you want to copy contact emails</p>'

In [4]: '<p class="help-block">%s</p>' % 'Useful when you want to copy contact emails'
Out[4]: '<p class="help-block">Useful when you want to copy contact emails</p>'
```